build/AzurePipelinesTemplates/WindowsAppSDK-SetupBuildEnvironment-Steps.yml

```diff
@@ -171,6 +171,14 @@ steps:
     arguments: -Path $(Build.SourcesDirectory)\dev\common\TerminalVelocityFeatures-PackageManager.xml -Channel $(channel) -Language C++ -Namespace Microsoft.Windows.Management.Deployment -Output $(Build.SourcesDirectory)\dev\common\TerminalVelocityFeatures-PackageManager.h
     workingDirectory: '$(Build.SourcesDirectory)'
 
+- task: powershell@2
+  displayName: 'Create CameraCaptureUI TerminalVelocity features'
+  inputs:
+    targetType: filePath
+    filePath: tools\TerminalVelocity\Generate-TerminalVelocityFeatures.ps1
+    arguments: -Path $(Build.SourcesDirectory)\dev\common\TerminalVelocityFeatures-CameraCaptureUI.xml -Channel $(channel) -Language C++ -Namespace Microsoft.Windows.Media.Capture -Output $(Build.SourcesDirectory)\dev\common\TerminalVelocityFeatures-CameraCaptureUI.h
```

Comment:
Why the generated '.h' file is being checked-in in the repo if you already have a build task to generate it?

Reply:
Just followed the documentation.
Without the header, the build would fail as we use it for feature check in implementation and test files.
My understanding is when the pipelines run, the file might get overwritten.

Reply:
Here, it seems a double work is happening. If build fails without header file, then what's the point of adding it in build-task.